package.json

```diff
@@ -38,7 +38,7 @@
     "@babel/preset-env": "^7.10.4",
     "@babel/preset-flow": "^7.10.4",
     "@babel/register": "^7.10.5",
-    "@playkit-js/playkit-js": "0.63.0",
+    "@playkit-js/playkit-js": "canary",
```

Comment:
can we make this explicitly done on the canary job only? add a step to install canary

Reply:
it's only for tests